LayoutTests/TestExpectations

```diff
@@ -8170,3 +8170,6 @@ imported/w3c/web-platform-tests/intersection-observer/cross-origin-tall-iframe-r
 imported/w3c/web-platform-tests/css/css-viewport/zoom/border-spacing.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-viewport/zoom/border-width.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-viewport/zoom/border.html [ ImageOnlyFailure ]
+
+# Hits assert in debug, tracked by rdar://165714112
```

Comment:
I think it would be nice for this go be a bugzilla link so that it's viewable to anyone!